opentutor_classifier/tests/fixtures/data/question3/config.yaml

```diff
@@ -0,0 +1,13 @@
+##
+## This software is Copyright ©️ 2020 The University of Southern California. All Rights Reserved. 
+## Permission to use, copy, modify, and distribute this software and its documentation for educational, research and non-profit purposes, without fee, and without a written agreement is hereby granted, provided that the above copyright notice and subject to the full license file found in the root of this software deliverable. Permission to make commercial use of this software may be obtained by contacting:  USC Stevens Center for Innovation University of Southern California 1150 S. Olive Street, Suite 2300, Los Angeles, CA 90115, USA Email: accounting@stevens.usc.edu
+##
+## The full terms of this copyright and license should always be found in the root directory of this software deliverable as "license.txt" and if these terms are not found with this software, please contact the USC Stevens Center for the full license.
+##
+question: "The dimensions of four rectangles are given above. Which rectangle do you think looks most like a square?"
+expectations:
+  - features:
+      good:
+        - "37\\s*(feet|ft)?\\s*(x|by)?\\s*40"
+      bad:
+        - "7\\s*(feet|ft)?\\s*(x|by)?\\s*10"
```

Comment:
so this is that thing where we're enumerating specific bad answers.

The ideal would be if we matched as `BAD` anything in the form 

`(<any-number-other-than-37>)(x|by)(<any-number-other-than-40>)`

?

...obviously the real regex above would be a lot more complicated.

I spent maybe 30 min trying to figure it out with negative lookahead, but no success so far